service/src/main/java/greencity/mapping/UserVOMapper.java

```diff
@@ -36,6 +38,23 @@ protected UserVO convert(User user) {
                     .build())
                 .collect(Collectors.toList());
         }
+
+        List<UserAchievementVO> userAchievements = new ArrayList<>();
+        if (user.getUserAchievements() != null) {
+            userAchievements = user.getUserAchievements()
+                .stream().map(userAchievement -> UserAchievementVO.builder()
```

Comment:
Move mapping operation to converter. And for VerifyEmailVO  and  for userFriends  also

Reply:
Done